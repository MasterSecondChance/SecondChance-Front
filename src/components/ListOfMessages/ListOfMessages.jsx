import React from 'react';
import Message from '../Message/Message';
import './ListOfMessages.scss';

const ListOfMessages = () => {
  return (
    <section className='ListOfMessages'>
      <Message />
      <Message />
      <Message />
      <Message />
      <Message />
      <Message />
      <Message />
      <Message />
      <Message />
      <Message />
    </section>
  );
};

export default ListOfMessages;
