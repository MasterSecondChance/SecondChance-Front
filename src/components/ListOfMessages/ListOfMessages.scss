@import '../../assets/styles/containers/Variables.scss';

.ListOfMessages {
  max-width: 500px;
  padding: 0 $spacing-02;
}

@media screen and (min-width: 768px) {
  .ListOfMessages {
    margin: $spacing-05 auto 0;
  }
}