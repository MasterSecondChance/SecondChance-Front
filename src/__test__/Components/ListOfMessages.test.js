import React from 'react';
import { mount, shallow } from 'enzyme';
import ProviderMock from '../../__mocks__/providerMock';
import ListOfMessages from '../../components/ListOfMessages/ListOfMessages';
import renderer, { create } from 'react-test-renderer';

describe('<ListOfMessages />', () => {
  test('Render del componente ListOfMessages', () => {
    const listofMessages = shallow(
      <ProviderMock>
        <ListOfMessages />
      </ProviderMock>
    );
    expect(listofMessages.length).toEqual(1);
  })
})

describe('ListOfMessages Snapshot', () => {
  test('Comprobar la UI del ListOfMessages', () => {
    const listofMessages = create(<ListOfMessages />)
    expect(listofMessages.toJSON()).toMatchSnapshot();
  })
})