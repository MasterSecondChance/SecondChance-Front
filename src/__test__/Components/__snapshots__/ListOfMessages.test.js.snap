// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListOfMessages Snapshot Comprobar la UI del ListOfMessages 1`] = `
<section
  className="ListOfMessages"
>
  <div
    className="Toastify"
  />
  <section
    className="ListOfMessages"
  />
</section>
`;
